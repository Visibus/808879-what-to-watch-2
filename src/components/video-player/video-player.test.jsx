import VideoPlayer from "./video-player";

it(`Is video player rendered`, () => {
  const tree = window.renderer
    .create(<VideoPlayer
      src={``}
      poster={`img/`}
      muted={true}
      width={280}
      height={175}
      isPlaying={{}}
    />);

  expect(tree).toMatchSnapshot();
});
