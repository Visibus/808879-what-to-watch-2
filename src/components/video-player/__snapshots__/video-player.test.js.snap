// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Is video player rendered 1`] = `
<video
  height={175}
  muted={true}
  poster="img/"
  src=""
  width={280}
/>
`;
